[DOCS] CLI Clean-up (#7904)
great-expectations/great_expectations: docs/docusaurus/docs/guides/validation/validation_actions/how_to_collect_openlineage_metadata_using_a_validation_action.md

### 3.  Test your Action by Validating a Batch of data.

> Run your Checkpoint to Validate a <TechnicalTag tag="batch" text="Batch" /> of data and emit lineage events to the OpenLineage backend. This Python code can retrieve and run a Checkpoint:

Reviewer: ```suggestion
Run the following command to retrieve and run a Checkpoint to Validate a <TechnicalTag tag="batch" text="Batch" /> of data and then emit lineage events to the OpenLineage backend:
```
